src/Symfony/Component/Debug/FatalErrorHandler/UndefinedMethodFatalErrorHandler.php

```diff
@@ -36,8 +36,12 @@ public function handleError(array $error, FatalErrorException $exception)
 
         $message = sprintf('Attempted to call an undefined method named "%s" of class "%s".', $methodName, $className);
 
+        if (null === $methods = get_class_methods($className)) {
```

Comment:
do we know any other condition that could make this return null? if not, it'd be more readable to use `class_exists`, don't you thing?